proto/desmos/profiles/v1beta1/packets.proto

```diff
@@ -0,0 +1,16 @@
+syntax = "proto3";
+package desmos.profiles.v1beta1;
+
+import "gogoproto/gogo.proto";
+import "desmos/profiles/v1beta1/msgs.proto";
+
+option go_package = "github.com/desmos-labs/desmos/x/profiles/types";
+
+// LinkChainAccountPacketData defines a struct for the packet payload
+message LinkChainAccountPacketData {
+  desmos.profiles.v1beta1.MsgLinkChainAccount msg = 1
+      [ (gogoproto.moretags) = "yaml:\"msg\"" ];
```

Comment:
I would vouch against including a message inside a packet data. Even thought it might seem counter-intuitive or a bad thing, I think we should simply copy/paste the same fields that `MsgLinkChainAccount` has inside the packet itself, in order to avoid having to create multiple objects just to send a packet: 
```suggestion
  string source_address = 1      [ (gogoproto.moretags) = "yaml:\"source_address\"" ];  
  desmos.profiles.v1beta1.Proof source_proof = 2      [ (gogoproto.moretags) = "yaml:\"source_proof\"" ];  
  desmos.profiles.v1beta1.ChainConfig source_chain_config = 3      [ (gogoproto.moretags) = "yaml:\"source_chain_config\"" ];  
  string destination_address = 4      [ (gogoproto.moretags) = "yaml:\"destination_address\"" ];  
  desmos.profiles.v1beta1.Proof destination_proof = 5      [ (gogoproto.moretags) = "yaml:\"destination_proof\"" ];
```